docs/common/craft-parts/how-to/use_parts.rst

```diff
@@ -0,0 +1,115 @@
+.. _how-to-use-parts:
+
+Using parts in a project file
+=============================
+
+When packing, :ref:`parts <parts>` are used to describe your package, where its various
+components can be found, its build and run-time requirements, and its dependencies.
+Consequently, all packages will have one or more parts.
+
+.. _how-to-use-parts_details:
+
+Parts are purposefully flexible to allow for varied and disparate sources. At its
+simplest, a part will locate a project's source code and invoke a
+:ref:`plugin <plugins>` to build the application in the environment. However, a part
+can just as easily be used to source and extract a binary executable from an RPM file.
+It can also download tagged code from a remote repository, pull in dependencies,
+define a build order, or completely override any stage of the
+:ref:`lifecycle <lifecycle>`.
+
+.. _how-to-use-parts_defining:
+
+Defining a part
```

Comment:
Having recently written a how-to, I'll share the feedback that I got that I think also applies to this page - a lot of the content here is more "reference" than "how-to". How-tos should be step-oriented, which you probably know as the paragraphs that start this section go in that direction and seem to be new (as far as I can tell).

In my case, I found it really helpful to do an outline of the sections/steps before-hand and getting an approval on that. Of course, @medubelko is the subject matter expert here and can (please) correct me if I'm wrong